src/connectors/sso.ts

```diff
@@ -37,6 +37,8 @@ function getQsParam(name: string) {
 
 function initiateBrowserSso(code: string, state: string) {
     window.postMessage({ command: 'authResult', code: code, state: state }, '*');
+    document.getElementById('content').innerHTML = '<p>Successfully passed off authentication to the extension.</p>' +
```

Comment:
We would need to figure out how to get the i18n service for localization in here to fetch this string value for multi-lingual support.

Also, perhaps we can re-word this to something like: 
> Authentication was successful. You may now close this tab and return to the browser extension.